docs/UserGuide.md

```diff
@@ -330,8 +330,8 @@ Format: `edit -e INDEX [d/DESCRIPTION] [at/DATE_TIME]
 * The index must be a positive integer 1, 2, 3, ...
 * At least one of the optional fields must be provided.
 * Existing values will be updated to the input values.
-* `[ap/CONTACT_INDEX_LIST]` adds contacts with the specified indexes to the event.
-* `[rp/ATTENDEE_INDEX_LIST]` removes contacts with the specified indexes from the event.
+* `[ap/CONTACT_INDEX_LIST]` add contacts with the specified indexes to the event.
+* `[rp/ATTENDEE_INDEX_LIST]` remove contacts with the specified indexes from the event.
```

Comment:
[field] _adds_, so i think these changes should be reverted.

Reply:
Did not think that the [field] should be treated as a noun. But it seems like the other parts of the UG do. I've reverted it back to follow other parts if the UG.